### ISSUE TITLE
[new] ambiguous getters calls

### ISSUE THREAD

**Issue by kalessil** [kalessil/phpinspectionsea#16]
Original link https://bitbucket.org/kalessil/phpinspectionsea/issues/135
Author @kalessil

Shall report duplicated sfContext::getInstance()->getLogger();

```php
sfContext::getInstance()->getLogger()->info(...);
sfContext::getInstance()->getLogger()->info(...);
```

**Comment by rentalhost** [comment]
Hard to implements. Except if it is especifically for symfony, what ensures that `getInstance()` or `getLogger()` will returns the same instance on both times?

I guess that it is possible after implements #221 (I am still working on it).

**Comment by Fleshgrinder** [comment]
I think that this could be a good sniff in general for getters. Multiple calls to getters are always prone to the problem that there is no guarantee that the same value is returned. Regardless of whether the call is static or dynamic. However, the consuming code usually assumes that exactly that is the case.

This sniff could be simplified to simply report multiple getter calls within a given routine.

**Comment by kalessil** [comment]
Sure, we are doing complex analysis (like this one) only for function/method scope.

In this case, we can only check sequential calls - should be enough to attract an attention to the routine.
